### DISCUSSION TITLE
Configure nearest ratio in the path-cheapest-arc strategy

### DISCUSSION THREAD

**Discussion by mohsenk** [google/or-tools#3466] [Routing (and legacy CP) questions]
Hi Guys
I'm testing or-tools to solve a VRP with one vehicle and time windows. ( without capacity and other constraints.)

I have tested multiple algorithm with over 1000 different VRP files. And i can say CHEAPEAST_ARC_PATH algorithm is the best in total.
But sometimes is not smart. and left the near stops and go do another stops.

For example in this VRP, `#95` and `#96` is on the way and it can easily do them.  And as you see it goes this way : `#17` , `#18` and ...

Expect to the final cost. how can i configure the algorithm to do all the stops that are close to each other. 

I found something called SavingNearestRation in the searchParameters that only works for the SAVING strategy. ( If i'm not mistaken )

![Screenshot from 2022-09-22 16-09-48](https://user-images.githubusercontent.com/1856478/191750455-960020cd-9a6a-48f8-84ed-d18ee65f6e14.png)

My configuration :

```
main.defaultRoutingSearchParameters()
                .toBuilder()
                .setLogSearch(true)
                .setLogTag("ROE-V2")
                .setSolutionLimit(10000)
                .setFirstSolutionStrategy(FirstSolutionStrategy.Value.PATH_CHEAPEST_ARC)
                .setLocalSearchMetaheuristic(LocalSearchMetaheuristic.Value.GUIDED_LOCAL_SEARCH)
                .setTimeLimit(Duration.newBuilder().setSeconds(30).build())
                .build();
```

**Comment by lperron** [comment]
It should be improved easily by local search.

**Reply by mohsenk** [reply]
Hi Leprron,
Thanks for the answer.
But how ? 
I'm using the GUIDED_LOCAL_SEARCH.
